mage.go

```diff
@@ -0,0 +1,49 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/sh"
+)
+
+const versionStringEnvVarName = "POCKET_VERSION_STRING"
+const pocketPackage = "github.com/pokt-network/pocket/cmd/pocket"
+const ldflag = "-X main.version=$" + versionStringEnvVarName
+
+var env = map[string]string{}
+
+func init() {
+	env["GO111MODULE"] = "on"
+
+	fmt.Println("Getting branch and commit information")
```

Comment:
NIT: I think we should use `log.Println` since eventually, we'll be passing a logger (with a bunch of bells and whistles on top) around.

Reply:
The `mage.go` file is not part of the project per se; it's a separate source that is executed a la `rake` or `make` by `mage`. I don't think this will grow to have a logger soon.